### PR TITLE
Handle multiple clusters

### PR DESCRIPTION
**Pull request by sarahhodne**
Add code to handle multiple clusters, since currently we can only handle one at a time.

Also includes a few more changes for consistency with other Go apps that we run, and also some more log messages.

### CODE REVIEW

**Review by meatballhat — APPROVED** [review]

File: cmd/collectd-vsphere/main.go [hunk 1]
```diff
@@ -49,42 +49,57 @@ func main() {
...
 			},
 			&cli.StringFlag{
 				Name:    "vsphere-cluster",
 				Usage:   "path to the vSphere cluster to monitor events on",
-				EnvVars: []string{"VSPHERE_CLUSTER"},
+				EnvVars: []string{"COLLECTD_VSPHERE_VSPHERE_CLUSTER", "VSPHERE_CLUSTER"},
+			},
+			&cli.StringSliceFlag{
+				Name:    "vsphere-clusters",
+				Usage:   "paths to the vSphere clusters to monitor events on",
```
Comment (igorwwwwwwwwwwwwwwwwwwww) [inline cmd/collectd-vsphere/main.go]: this should explain how you pass more than one. does the user pass the flag more than once? or is it comma-separated?

File: cmd/collectd-vsphere/main.go [hunk 2]
```diff
@@ -116,21 +132,39 @@ func mainAction(c *cli.Context) error {
...
-		BaseVMPath:  c.String("vsphere-base-vm-folder"),
-	}, statsCollector)
+		URL:          u,
+		Insecure:     c.Bool("vsphere-insecure"),
+		ClusterPaths: clusterPaths,
+		BaseVMPaths:  baseVMPaths,
+	}, statsCollector, logger.WithField("component", "vsphere-event-listener"))
 
-	panicErr, _ := raven.CapturePanic(func() {
+	panicErr, _ := raven.CapturePanicAndWait(func() {
```
Comment (igorwwwwwwwwwwwwwwwwwwww) [inline cmd/collectd-vsphere/main.go:167]: 👍

**Review by igorwwwwwwwwwwwwwwwwwwww — APPROVED** [review]
I skimmed over some of the messy details in `vsphere_event_listener.go` since I'm not super familiar with that code yet, but I'd say LGTM! 👍

File: cmd/collectd-vsphere/main.go (hunk 1 above)
Comment (sarahhodne) [reply cmd/collectd-vsphere/main.go]: Good point, it's comma-separated, but it doesn't look like that shows in the help text at all:

```
   --vsphere-clusters value          paths to the vSphere clusters to monitor events on [$COLLECTD_VSPHERE_VSPHERE_CLUSTERS, $VSPHERE_CLUSTERS]
```

How about changing to

``` Go
Usage: "comma-separated paths to the vSphere…"
```

(and same for the other slice flags)?

Comment (igorwwwwwwwwwwwwwwwwwwww) [reply cmd/collectd-vsphere/main.go]: 👍

**Pull request merged**